nvflare/fuel/hci/client/api.py

```diff
@@ -164,6 +165,28 @@ def _load_client_cmds(self, cmd_modules):
     def register_command(self, cmd_entry):
         self.all_cmds.append(cmd_entry.name)
 
+    def session_monitor(self, session_callback):
+        _session_monitor_thread = threading.Thread(target=self._check_session, args=(session_callback,), daemon=True)
+        _session_monitor_thread.start()
```

Comment:
If the api is a long lived object in the system,
after someone called api.session_monitor() it will start this session monitor thread,
I think we would expect there is a method that can end / join this thread back, rather than close this thread when the whole process end?

Reply:
Yes I definitely agree. I just spoke with Yan about this as well.